hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -165,6 +168,12 @@ public Operation convert(String value) throws ParameterException {
     }
   }
 
+  private static final class VoidSplitter implements IParameterSplitter {
```

Comment:
How about make this class public and move it to a more general package e.g. ```hudi-common```?
There are some other classes which might also need this ```VoidSplitter```, e.g. https://github.com/apache/hudi/blob/master/hudi-utilities/src/main/java/org/apache/hudi/utilities/HDFSParquetImporter.java#L280, https://github.com/apache/hudi/blob/master/hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieCompactor.java#L92

Reply:
I was a little skeptical about making this change apply for all CLIs. But on second thought, looked at all documentation and README files all of them have `--hoodie-conf A --hoodie-conf B` way so no one in theory should be sending `--hoodie-conf A,B` in their usecases. So will move it to common and apply this change for all of the classes using hoodie-conf

Reply:
Thanks for confirming!

Reply:
@zhedoubushishi : Lets not bring in jcommander dependency to hudi-common. hudi-common is a core package used across different environments. This can be made the first level class but lets keep it in hudi-utilities : org.apache.hudi.utilities